hype-submitter_2.11/src/main/scala/com/spotify/hype/GkeCluster.scala

```diff
@@ -0,0 +1,16 @@
+package com.spotify.hype
+
+
+case class GkeCluster(project: String,
+                      zone: String,
+                      cluster: String,
+                      staging: String
```

Comment:
Right, I asked myself this question. The alternative is to keep two objects (on GkeCluster and one hype Submitter). I believe it's rare that someone would use different submitters for the same cluster.
Since these objects are quite related, it would be more simple for the API to build 1 object rather than two, although staging is not, per say part of the cluster. It's usability/transparency tradeoff at the end of the day.